api/src/lib/knex.js

```diff
@@ -1,6 +1,6 @@
 var config = require('../../config');
 var knex = require('knex')({
-  debug : process.env.DEBUG,
+  debug : false, //process.env.DEBUG,
```

Comment:
On purpose?

Reply:
I'm not opposed if it is.

Reply:
not on purpose but i'm not worried about merging that in